src/components/Searcher.js

```diff
@@ -204,6 +207,7 @@ type SearcherState = {
   resultsPerPage: number;
   resultsOffset: number;
   debug: boolean;
+  queryTimestamp: number;
```

Comment:
I don't think we still need this if you're using the signal from the search results.